app/views/layouts/landing.html.erb

```diff
@@ -24,7 +24,8 @@
     <%= discourse_stylesheet_link_tag "landing_page", theme_id: nil %>
 
     <%- if theme_id.present? %>
-      <%= discourse_stylesheet_link_tag (mobile_view? ? :mobile_theme : :desktop_theme), theme_id: theme_id %>
+      <%= discourse_color_scheme_stylesheets %>
+      <%= discourse_stylesheet_link_tag (mobile_view? ? :mobile : :desktop), theme_id: theme_id %>
```

Comment:
@cabidop Whether it's this change or some other changes, page styling of non-inline pages seems to be broken. For example if you import pages from https://github.com/paviliondev/pavilion-landing-pages and apply the theme https://github.com/paviliondev/pavilion-landing-theme you see this

<img width="1470" alt="Screenshot 2023-09-19 at 10 21 45" src="https://github.com/paviliondev/discourse-landing-pages/assets/5931623/b90c0f24-4b28-4398-afff-03e11cbaa216">

Reply:
Absolutely right, seems that both tags are required as the `:mobile/:desktop` is used for the base styles and the `:mobile_theme/:desktop_theme` is used for the imported themes and the "manual" updates on the default one.

Reply:
Also the current code it is actually ignoring the page setting for the theme, as all the conditionals use `theme_id.*` instead of `@page.theme_id.*`.